docker/consumer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16-alpine
+
+ENV GO111MODULE=on
+WORKDIR /app
+
+COPY . .
+
```

Comment:
For the starter project this is fine, but you might want to look at running with less privileges. (Have a look at some of the other services)